content/ja/handbook/leadership/index.md

```
@@ -0,0 +1,333 @@
+
+[コーチング](/handbook/leadership/coaching/)は他者が自身を助けるのを支援することです。アドバイスを与えたり、指示したり、何をすべきかを伝えることではありません。**コーチングは将来に焦点を当て、コーチーが目指す場所と達成したいことを特定することについてです。** GitLabでは、コーチングを人々が自分で考え、自分自身の答えを見つけ、自分で設計したアクションにコミットするのを助ける会話として定義しています。コーチとして、あなたの役割は現在の状態から将来への道筋を明確にすることです。コーチはより深い洞察に基づいたインフォームドな選択をコーチーが行えるようにすることでこれを実現します。
+
+## マトリクス組織なし
+
+[no-matrix-organization](/handbook/leadership/no-matrix-organization/)をご覧ください。
+
+## 安定したカウンターパート
+
+機能横断的なコラボレーションを有機的に促進するために、他の機能で一緒に働く必要がある人々に安定したカウンターパートを与えることを望んでいます。例えば、各Strategic Account Executive（SAE）は一人のSales Development Representative（SDR）と働きます。[カテゴリー](/handbook/product/categories/)では、すべてのバックエンド開発者チームが[Product Manager（PM）](/job-description-library/product/product-manager/)と[フロントエンドチーム](/handbook/engineering/)にマッピングされています。
+
+人々に**安定したカウンターパート**を与えることで、より社会的信頼と親しみが生まれ、意思決定が加速し、コミュニケーションの問題を防ぎ、コンフリクトのリスクを減らします。このようにして、[マトリクス組織のデメリット](#no-matrix-organization)なしに機能横断的に効果的に作業できます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)